system/netdata.service.in

```diff
@@ -1,15 +1,56 @@
+# netdata systemd target
+
 [Unit]
-Description=Real time performance monitoring
+Description=netdata - Real-time performance monitoring
```

Comment:
I've noticed that some daemons (tinydns, etcd, openvpn, ...) have their name in the description. I can remove it if you prefer.